extensions/wikia/AdEngine/js/provider/evolve2.js

```diff
@@ -102,6 +109,11 @@ define('ext.wikia.adEngine.provider.evolve2', [
 		log(['fillInSlot', slot.name, 'done'], 'debug', logGroup);
 	}
 
+	resetPosTargeting();
```

Comment:
I'm thinking if there is better place for this line and 3 below... More cleaner way to do it. It's a minor note - no action needs to be taken. Yet ;)